### DISCUSSION TITLE
how to manage assets folder in alternative way?

### DISCUSSION THREAD

**Discussion by CanYellow** [requarks/wiki#6785] [Help / Questions]
dear my friends,
It is said that manage assets folder is still not supported in wikijs 2 doc.

But I initially create some asset folder for testing my installation and get familiar to the wiki system.

So is there alternative way to delete the folder? for example, just deal with the os file or database record?

If so, could you give me some suggestion?

thanks!

**Comment by P4mes4n** [comment]
You can delete asset folders by interacting with the Database.
To liste all folders with IDs:
`SELECT * FROM "assetFolders";`
To delete folder with ID XX:
`DELETE FROM "assetFolders" WHERE id=XX;`

**Comment by CanYellow** [comment]
Thank you very much!  The problem was solved according to your advice.

and here is the process.

after the question, I alreadly logined in my database and found the 'assetFolders' with \d command. but when I used '\d assetFolders' , it throe a error:  did not find ...
this is where I stoped.

update solution.

first, the correct command above is '\d "assetFolders"' with double quotation mark.

and just now, following you your instruction above, I already delete the folder with ID. Thank you again.
